xil-Town05/cdasim_config/evc_sumo/evc_sumo_cfg.json

```diff
@@ -1,20 +1,56 @@
 {
-    "controllers": [
+  "controllers": [
+  
+  
+       {
+      "controllerId": 1,
+      "controllerCfgPath": "/home/carma/src/resources/test_3.cfg",
```

Comment:
would it be possible to rename evc config .cfg file to be the same with sumoTlId, such as 421.cfg